calico-policies/public-network-isolation/eu-es/allow-public-services-pods.yaml

```diff
@@ -0,0 +1,87 @@
+# This policy allows outgoing network traffic from pods to other IBM Cloud services.
+# The example nets and ports in this policy are for IBM Cloud Container Registry, IBM Cloud
+# Monitoring with Sysdig, IBM Log Analysis with LogDNA, and the managed Istio add-on.
+
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: allow-public-services-pods
+spec:
+  egress:
+  - action: Allow
+    destination:
+      nets:
+      # IBM Cloud Container Registry: Global registry
+      - 169.60.98.80/29
+      - 169.62.37.240/29
+      - 169.63.104.232/29
+      - 169.47.153.160/28
+      - 103.41.69.127/32
+      - 103.41.69.114/32
+      - 103.41.69.115/32 
+      # IBM Cloud Container Registry: EU Central nets for your own registry
+      - 149.81.79.152/29
+      - 161.156.93.16/29
+      - 169.50.58.104/29
+      - 103.41.70.147/32
+      - 103.41.70.149/32
+      - 103.41.70.150/32
+      - 103.41.70.153/32
+      - 103.41.70.154/32
+      - 103.41.70.156/32
+      ports:
+      - 443
+      - 4443
+    protocol: TCP
+    source: {}
+  - action: Allow
+    destination:
+      nets:
+      # IBM Cloud Monitoring with Sysdig: eu-de
+      # Taken from: https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints
+      - 149.81.77.78/32
```

Comment:
Do you know if sysdig and/or logDNA have (or will have) separate IPs for eu-es (separate from the eu-de ones)? Just because registry doesn't have separate eu-es IPs, doesn't mean logDNA or sysdig will necessarily do the same thing.

Reply:
Yes, I confirmed this as of now they are using eu-de endpoints for logDNA and Sysdig and will change in the future.